_notebooks/2021-05-04-spectral-svd.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p>Titles should be something more like ASE (graspologic vs scikit-network)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neurodata/notebooks/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>